src/commands/roles.ts

```diff
@@ -45,7 +45,9 @@ export default class extends BotCommand {
 			!msg.guild ||
 			msg.guild.id !== SupportServer ||
 			!role ||
-			['342983479501389826', Roles.Moderator, '590527706974781470', '346238402737340416'].includes(role.id) ||
+			// 346238402737340416 = Bot Role
+			// 590527706974781470 = Skira user?
+			[Roles.Moderator, '590527706974781470', '346238402737340416'].includes(role.id) ||
```

Comment:
These comments are really weird.

Reply:
I thought the same thing. I have no idea why skyra is there, but at least we know what that means. I have no idea if that means skyra user or some sort of hidden role.